CDC-Data-Reconciliation-Backend/cliComparison.py

```diff
@@ -50,11 +50,12 @@ def main():
 
     parser = argparse.ArgumentParser(
         prog="AutoCompare", description='Auto Comparison')
-    parser.add_argument('-c', '--cdc', help='Local Path to CDC CSV file')
-    parser.add_argument('-o', '--output', help='Name of folder that should be created to store the report files')
-    parser.add_argument('-y', '--year', help='Year to compare')
-    # if the parameter below is specified the value stored is true
-    parser.add_argument('-f', '--filter', action='store_true', help='Filter by CDC eventCodes')
+    parser.add_argument('-c', '--cdc', required=True, help='Local Path to CDC CSV file')
+    parser.add_argument('-o', '--output', required=True, help='Name of folder that should be created to store the report files')
+    parser.add_argument('-y', '--year', required=True, help='Year to compare')
+    # defaulting to filtering by CDC event codes
+    # to disable filtering, add -f False (or some other falsy value - e.g., '0', 'f', 'n')
+    parser.add_argument('-f', '--filter', type=lambda x: (str(x).lower() in ['true', '1', 't', 'y', 'yes']), default=True, help='Filter by CDC eventCodes')
```

Comment:
I feel like this argument doesn't make a whole lot of sense as someone can type "-f fjdaklfdsk" and the filter will be set to false. Maybe we can set this to "-nf" and "--nofilter"? Then have the default be False and the action as "store_true". Then on line 74, have filterByCDC = !args.filter